amplify/src/main/java/com/github/stkent/amplify/utils/FeedbackUtil.java

```diff
@@ -116,11 +124,16 @@ private String getApplicationInfoString() {
                 + "\n"
                 + "Android OS Version: " + getAndroidOsVersionDisplayString()
                 + "\n"
-                + "Date: " + dateString;
+                + "Date: " + getDateString();
+    }
+
+    private String getDateString() {
+        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("MM/dd/yyyy kk:mm:ss", Locale.getDefault());
+        Date date = new Date(SystemTimeUtil.currentTimeMillis());
+        return simpleDateFormat.format(date);
     }
 
     private String getAndroidOsVersionDisplayString() {
-        // fixme: will this correctly reporting version information for the consuming application??
```

Comment:
Yup. Well I looked at the email in the application I'm using it in and it displays the version information from my app not from Amplify.